development/cpp/configuring_an_ide.rst

```diff
@@ -296,3 +297,22 @@ To create them:
 - You can now start Debug process again to test that everything works.
 
 - If the build phase fails, check the console for hints. On Linux it's most likely that some dependencies are missing. Check :ref:`Compiling for X11 (Linux, \*BSD) <doc_compiling_for_x11>`
+
+
+.. _doc_configuring_an_ide_android_studio:
+
+Android Studio
+-----------------
```

Comment:
reST is quite opinionated, it wants underlines to match the exact character count of the headings. So there are a few too many here.